src/systems/ec/implementations/react/components/date-block/src/DateBlock.jsx

```diff
@@ -16,26 +16,26 @@ const DateBlock = ({
 
   return (
     <div {...props} className={classNames}>
```

Comment:
it would be nice to use a `<time>` tag with the `datetime` attribute and then use `<span>`s  instead of `<div>`s  inside